accounts/views.py

```diff
@@ -73,6 +76,81 @@ def citesphere_token(request):
         token.save()
         return Response(status=status.HTTP_201_CREATED)
 
+class ForgotPasswordView(APIView):
+    def post(self, request):
+        username = request.data.get('username', None)
+        if username:
+            try:
+                user = VogonUser.objects.get(username=username)
+            except VogonUser.DoesNotExist:
+                return Response({ "success": False, "message": "User not found" }, status=404)
+            token = jwt.encode({ 
+                'exp': arrow.utcnow().shift(days=1).timestamp,
+                'username': user.username,
+            }, settings.SECRET_KEY, algorithm='HS256').decode('UTF-8')
```

Comment:
why do you encode the secret key here?

Reply:
JWT uses secret key to encode the contents of the dict.